priv/kafka.bnf

```diff
@@ -250,3 +250,9 @@ DescribeGroupsResponse => [GroupDescription]
     MemberMetadata => bytes
     MemberAssignment => bytes
 
+SaslHandshakeRequest => Mechanism
+  Mechanism => string
+
+SaslHandshakeResponse => ErrorCode [EnabledMechanism]
+  ErrorCode => int16
+  EnabledMechanism => string
```

Comment:
You are right, I think it's a typo or a mistake in kafka protocol specification.

Please don't change it!